addons/stock/tests/test_stock_return_picking.py

```diff
@@ -209,3 +209,78 @@ def test_stock_return_for_exchange(self):
         return_picking = self.env['stock.picking'].browse(res['res_id'])
         self.assertTrue(return_picking)
         self.assertEqual(len(return_picking.move_ids), 1)
+
+    def test_stock_picking_report_has_return(self):
+        """
+        Test that the stock.lot.report has_return field is correctly set by simulating a 
+        2-product delivery (2 serial numbers) and a return of 1 product.
+        Expected:
+        - The report should contain 2 lines.
+        - One line has has_return = True.
+        - One line has has_return = False.
```

Comment:
Nice to always add a docstring! :ok_hand:  
Could you make it more "business" ? Like a short summary of the use case and the expected behaviour ?